<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="202150" started-at="2020-01-05T15:02:03Z" finished-at="2020-01-05T15:05:26Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="202150" started-at="2020-01-05T15:02:03Z" finished-at="2020-01-05T15:05:26Z">
      <class name="automationFramework.automationFramework.FirstTestCaseTest">
        <test-method status="PASS" signature="beforeTest()[pri:0, instance:automationFramework.automationFramework.FirstTestCaseTest@d2cc05a]" name="beforeTest" is-config="true" duration-ms="113740" started-at="2020-01-05T15:02:04Z" finished-at="2020-01-05T15:03:57Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="FAIL" signature="testTask()[pri:0, instance:automationFramework.automationFramework.FirstTestCaseTest@d2cc05a]" name="testTask" duration-ms="85208" started-at="2020-01-05T15:03:57Z" finished-at="2020-01-05T15:05:23Z">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[Expected condition failed: waiting for presence of element located by: By.className: btn--honest blattgold--form-banner-submit (tried for 30 second(s) with 500 milliseconds interval)]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: Expected condition failed: waiting for presence of element located by: By.className: btn--honest blattgold--form-banner-submit (tried for 30 second(s) with 500 milliseconds interval)
at automationFramework.automationFramework.FirstTestCaseTest.testTask(FirstTestCaseTest.java:38)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testTask -->
        <test-method status="PASS" signature="afterTest()[pri:0, instance:automationFramework.automationFramework.FirstTestCaseTest@d2cc05a]" name="afterTest" is-config="true" duration-ms="3072" started-at="2020-01-05T15:05:23Z" finished-at="2020-01-05T15:05:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
      </class> <!-- automationFramework.automationFramework.FirstTestCaseTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
